button/black.js

```diff
@@ -0,0 +1,13 @@
+import { html } from '../template.js';
+
+export const black = html`
+<style>
```

Comment:
You can import button here and inject it with the style, so that people importing black have all the style necessary:
```js
import { button } from './button.js'

export const black = html`
${button}
<style>
...
</style>
`

```